README.md

```diff
@@ -1 +1,233 @@
+    <li><a href="">Html</a></li>
+    <li><a href="">Css</a></li>
+    <li><a href="">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="">n/a</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="/">n/a</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[use linters]**
+- **[Hero section]**
+- **[Course lists sections]**
+- **[Our team section]**
+- **[Our partners section]**
+- **[Latest studend projects done ]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ <p><a href="https://zbahati.github.io/web-development-course/">Online Web development Demo </a></p>
+
+ <p><a href="https://www.loom.com/share/82c4ab60f3e44957b7ff351bfa7c1742">Use this Demo Video to introducing the project in General</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- you need a laptop.
+- you need github installed on it.
+- You need a basic knowledge of Html and Css and JavaScript
+
+
+### Setup
+
+Clone this repository to your desired folder:
+- cd my-folder
+- git clone https://github.com/zbahati/web-development-course.git
+
+
+### Install
+
+Install this project with:
+
+
+- cd my-project
+- npm install .
+
+
+### Usage
+
+To run the project, execute the following command:
+- open your local server on your computer
+
+
+### Run tests
+
+To run tests, run the following command:
+
+- npx stylelint "**/*.{css,scss}"
+- npx hint .
+- npx eslint .
+
+### Deployment
+
+You can deploy this project using: github page's
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Bahati zirimwabagabo**
+
+- GitHub: [zbahati](https://github.com/zbahati)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/zirimwabagabo-bahati)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Nav menu toggle on mobile devices]**
+- [ ] **[Add server]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+show me your support by following me on github and giving me a star please .
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse.
```

Comment:
I see why you'd want to thank Microverse 😉 they're doing a great job, but make sure that the README also contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 🔗

Reply:
i Fixed also the acknowlegment where i send my acknowledgment to the author for this original design